pandas/tests/resample/test_deprecated.py

```diff
@@ -43,20 +43,20 @@ def test_deprecating_on_loffset_and_base():
     idx = pd.date_range("2001-01-01", periods=4, freq="T")
     df = DataFrame(data=4 * [range(2)], index=idx, columns=["a", "b"])
 
-    with tm.assert_produces_warning(FutureWarning):
+    with tm.assert_produces_warning(FutureWarning, check_stacklevel=False):
```

Comment:
Those changes can be removed again?

Reply:
The whole point of this PR is to remove those checks :)

Reply:
> The whole point of this PR is to remove those checks

But above you say that you restored them?

Reply:
I realise I might have sounded confusing: I restored the code that _satisfied_ these checks, but I haven't restored the checks themselves.